documentation/asciidoc/computers/configuration/device-tree.adoc

```diff
@@ -859,9 +866,17 @@ Each property is stored as a 32-bit integer unless indicated otherwise.
 
 `tryboot`:: Set to `1` if the `tryboot` flag was set at boot.
 
+==== Boot variables `/chosen/bootloader`
+
+Raspberry Pi 5 only. Each property is stored as a 32-bit unsigned integer unless indicated otherwise.
```

Comment:
Another small nitpick (sorry) but should it say "Each variable is...", given that the header is "Boot variables" ?

Reply:
That would be inconsistent with the other sections of device-tree properties i.e. we are talking about a device-tree property which encapsulates a boot-variable.

However, given that "Each property is stored as a 32-bit variable" is repeated throughout the entire firmware properties section I've hoisted that to the top and removed the duplication